packages/app-page-builder/src/editor/components/Element.tsx

```diff
@@ -139,6 +139,7 @@ const ElementComponent: React.FC<ElementPropsType> = ({
                     active={isActive || false}
                     style={{ ...defaultStyle, ...transitionStyles[state] }}
                     className={"webiny-pb-page-element-container"}
+                    data-testid={"pb-page-element"}
```

Comment:
```suggestion
                    data-testid={"page-element"}
```

I'd either loose the `pb-` prefix or update other test IDs and and it there as well.